spec-tests/src/main/java/org/candlepin/spec/bootstrap/client/SpecTestFixture.java

```diff
@@ -41,6 +45,19 @@ protected <O> O getClient(Class<O> outputClass) {
         return client;
     }
 
+    /**
+     * Retrieves if Candlepin is running in standalone mode or not.
+     *
+     * @return true if Candlepin is in standalone mode.
+     * @throws ApiException when unable to retrieve the status.
+     */
+    protected boolean isStandalone() throws ApiException {
+        ApiClient client = ApiClients.admin();
+        StatusApi statusApi = client.status();
+        StatusDTO status = statusApi.status();
+        return status.getStandalone();
+    }
```

Comment:
I'm not sure if this is the best place for this method. I'm assuming that there are other spec tests that will be dictated on whether or not Candlepin is running in standalone mode or not.

Reply:
Looks like @Januson also wrote this as two annotations we can set on methods/classes: @OnlyInHosted and @OnlyInStandalone. Perhaps we keep that?

Reply:
Yeah, I like the use of the annotations a lot more!

Reply:
This PR will be probably merged sooner that the one for owner spec. Once it's merged I'll rebase and add the annotation.

Reply:
@Januson  Awesome. I left a TODO comment on the test and commented out the test annotation.